test/e2e/basic_test.go

```diff
@@ -81,8 +81,12 @@ func (data *TestData) testDeletePod(t *testing.T, podName string, nodeName strin
 		t.Fatalf("Error when retrieving the name of the Antrea Pod running on Node '%s': %v", nodeName, err)
 	}
 	t.Logf("The Antrea Pod for Node '%s' is '%s'", nodeName, antreaPodName)
-
 	cmds := []string{"antctl", "get", "podinterface", podName, "-n", testNamespace, "-o", "json"}
+	if testOptions.enableCoverage {
+		const timeFormat = "20060102T150405Z0700"
+		timeStamp := time.Now().Format(timeFormat)
```

Comment:
can we generate the coverage .out file using a function, to avoid repeating this multiple times?

Reply:
use a function to generate the coverage .out file.